// @kintone/rest-api-client/esm/platform/index.js
export var platformDeps = {
    readFileFromPath: function () {
        throw new Error("not implemented");
    },
    getRequestToken: function () {
        throw new Error("not implemented");
    },
    getDefaultAuth: function () {
        throw new Error("not implemented");
    },
    buildPlatformDependentConfig: function () {
        throw new Error("not implemented");
    },
    buildHeaders: function () {
        throw new Error("not implemented");
    },
    buildFormDataValue: function () {
        throw new Error("not implemented");
    },
    buildBaseUrl: function () {
        throw new Error("not implemented");
    },
    getVersion: function () {
        throw new Error("not implemented");
    },
};
export var injectPlatformDeps = function (deps) {
    platformDeps.readFileFromPath = deps.readFileFromPath;
    platformDeps.getRequestToken = deps.getRequestToken;
    platformDeps.getDefaultAuth = deps.getDefaultAuth;
    platformDeps.buildPlatformDependentConfig =
        deps.buildPlatformDependentConfig;
    platformDeps.buildHeaders = deps.buildHeaders;
    platformDeps.buildFormDataValue = deps.buildFormDataValue;
    platformDeps.buildBaseUrl = deps.buildBaseUrl;
    platformDeps.getVersion = deps.getVersion;
};
